hypothesis-python/tests/nocover/test_conjecture_engine.py

```diff
@@ -255,7 +261,8 @@ def shrinker(data):
     def blocks(intervene=False):
         blocks = []
         for a, b in shrinker.each_pair_of_blocks(
-            lambda block: True, lambda block: True
+            lambda block: block.index not in exclude_first,  # non-trivial lambda
+            lambda block: block.index not in exclude_second,  # non-trivial lambda
```

Comment:
These comments were just to indicate where I had modified the test, so they can safely be removed.